components/hab/src/command/pkg/demote.rs

```diff
@@ -44,17 +44,17 @@ pub fn start(
     ui: &mut UI,
     bldr_url: &str,
     ident: &PackageIdent,
-    channel: &str,
+    channel: &ChannelIdent,
     token: &str,
 ) -> Result<()> {
     let api_client = Client::new(bldr_url, PRODUCT, VERSION, None)?;
 
     ui.begin(format!("Demoting {} from {}", ident, channel))?;
 
-    if channel == "unstable" {
+    if channel == &ChannelIdent::unstable() {
         return Err(Error::CannotRemoveFromChannel((
             format!("{}", ident),
```

Comment:
Might want to change this to `ident.to_string()`... odd that Clippy isn't flagging that 🤔

Reply:
My guess is that it has to do with crossing the crate boundary, but good catch, will fix.